src/flows/shared/steps/OrganizeCollection/Sidebar/Sidebar.tsx

```diff
@@ -199,11 +152,7 @@ const Header = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: baseline;
-`;
-
-const StyledSelectButtonWrapper = styled.div`
-  display: flex;
-  justify-content: flex-end;
+  min-height: 52px;
```

Comment:
looks like the min-height here might give a bit too much spacing between the header and searcher.

spec:
<img width="458" alt="Screen Shot 2022-07-06 at 13 30 42" src="https://user-images.githubusercontent.com/80802871/177550632-09f63b66-ade8-4d67-87fb-53fdf55cf85d.png">

dev:
<img width="250" alt="Screen Shot 2022-07-06 at 13 30 32" src="https://user-images.githubusercontent.com/80802871/177550640-0f95b8c4-a6de-4003-9a97-340129a7c511.png">

Reply:
Good catch, the sidebar itself has 16px of padding, so I've made the min-height of the text `calc(52px - 16px)` 

### Updated

<div style="display: flex;">
<img width="49%" alt="image" src=https://user-images.githubusercontent.com/80802871/177550632-09f63b66-ade8-4d67-87fb-53fdf55cf85d.png />

<img width="49%" alt="image" src="https://user-images.githubusercontent.com/13339581/177587486-53ed66c0-149a-49a5-8377-57fa4c563264.png">
</div>